internal/errors/cassandra.go

```diff
@@ -30,8 +30,25 @@ var (
 		}
 	}
 
-	ErrCassandraNotFound = func(key string) error {
-		return Wrapf(NewErrCassandraNotFoundIdentity(), "error cassandra key '%s' not found", key)
+	NewErrCassandraUnavailableIdentity = func() error {
+		return &ErrCassandraUnavailableIdentity{
+			err: New("error cassandra unavailable"),
+		}
+	}
+
+	ErrCassandraUnavailable = func() error {
+		return NewErrCassandraUnavailableIdentity()
+	}
+
+	ErrCassandraNotFound = func(keys ...string) error {
+		switch {
+		case len(keys) == 1:
+			return Wrapf(NewErrCassandraNotFoundIdentity(), "error cassandra key '%s' not found", keys[0])
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg?style=flat-square)
> "error cassandra key '%s' not found"

"error" is useless.
it's better to write just "cassandra key '%s' not found".
It's my mistake.